cli/setup.go

```diff
@@ -106,8 +106,11 @@ const (
 	promptDoneRunDaemon = "Starting Mender client…\n" + // NOTE: format
 		"Your device should show up at %s within the next few " +
 		"minutes.\nLog in to your Mender UI at %s to accept it.\n"
-	promptDeviceType         = "Device type (for example raspberrypi3-raspbian): "
-	promptConfirmDev         = "Confirm devicetype as %s? [Y/n] " // NOTE: format
+	promptDeviceType = "\nThe device type property is used to determine " +
+		"which mender-artifacts are compatible with this device.\n" +
```

Comment:
In docs and help text, we refer to them as "Mender Artifacts". I have never seen mender-artifact for anything but the standalone tool name